packages/flutter/lib/src/cupertino/activity_indicator.dart

```diff
@@ -117,8 +124,7 @@ class _CupertinoActivityIndicatorState extends State<CupertinoActivityIndicator>
       child: CustomPaint(
         painter: _CupertinoActivityIndicatorPainter(
           position: _controller,
-          activeColor:
-              CupertinoDynamicColor.resolve(_kActiveTickColor, context),
+          activeColor: widget.color ?? _kActiveTickColor,
```

Comment:
I think it's still necessary to resolve the color there before passing it as `activeColor`?

Reply:
Missed that, thanks